install-deps.sh

```diff
@@ -31,7 +31,7 @@ fi
 
 function install_seastar_deps {
     if $with_seastar; then
```

Comment:
Should this be capitalised for consistency?

Reply:
Ignore this, my bad. I see the relevant section where WITH_SEASTAR gets checked earlier in the file.

Reply:
@badone either way. i guess it could be simpler if we just have

```sh
if [ -n "$WITH_SEASTAR"] ; 
```

in this function instead of checking the two variables in different places? what do you think?

Reply:
Sounds like a good idea to me.